shared/versions/stack/7.13.asciidoc

```diff
@@ -28,6 +28,7 @@ is-current-version can be: true | false
 APM Agent versions
 ////
 :apm-go-branch:         1.x
+:apm-ios-branch:        main
```

Comment:
@bryce-b these files are used to map links from stack-versioned docs to the agent documentation. These branches represent the version of the documentation linked to from the stack.

As an example, this file is named 7.13.asciidoc. So any links from 7.13 stack docs will go to the main branch of ios, or the 1.x branch of the go agent

Reply:
gotcha! thanks for the clarification.